src/test/java/HomeWork18.java

```diff
@@ -0,0 +1,48 @@
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+public class HomeWork18 extends BaseTest{
+
+    @Test
+
+    public void playSongTest() throws InterruptedException {
+
+        navigateToPage();
+        provideEmail("guadalupe.medina@testpro.io");
+        providePassword("DoingitBig23!");
+        submitButton();
+
+
+    }
+    public void playSong(){
+            WebElement playButton = driver.findElement(By.cssSelector("span[@data-testid='play-btn']"));
+            WebElement playNextSongButton = driver.findElement(By.cssSelector("i[@data-testid='play-next-btn']"));
+
+            playButton.click();
+            playNextSongButton.click();
+
+            Assert.assertTrue(isSongPlaying());
+
+        }
+
+
+        public boolean isSongPlaying() {
+            WebElement soundBarPlay = driver.findElement(By.cssSelector("div[@data-testid ='sound-bar-play']"));
+            return soundBarPlay.isDisplayed();
+
+
+
+        }
+
+
+
+
+
+
+
+
```

Comment:
**Please delete the empty line!** An Empty Line is a "Code Smell"
The short answer is, _They’re more an annoyance than a problem.They won’t actually break anything—which is why your code continues to run—but they make it awkward to read and change your code, especially if you’re dealing with multiple line breaks per line of real code. They also add just a tiny bit of size to your files, which is nice to reduce._